src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs

```diff
@@ -3337,6 +3337,7 @@ public override void PostVisit(DottedNameNode node)
                 var nameRhs = node.Right.Name;
 
                 nameRhs = GetLogicalNodeNameAndUpdateDisplayNames(leftType, node.Right);
+                var nameRhsString = nameRhs.ToString();
```

Comment:
same here, nameRhs.Value

Reply:
Also, consider using node.Right.Name.Value instead of the logicalName that you get back from this function. The error message will be better if it corresponds to what's in the expression rather than the names under the hood

Reply:
Done